src/model/User.js

```diff
@@ -0,0 +1,29 @@
+const mongoose = require('mongoose');
+const Promise = require('bluebird');
+
+const { Schema } = mongoose;
+const UserSchema = new Schema({
+	id: Number,
+	token: String,
+	email: String,
+	name: String,
+	provider: String,
+});
+
+UserSchema.statics = {
```

Comment:
why this is needed in the model that can be kept in the crud operations

Reply:
If we are moving crud at one place then we can take this code away from here.